docs/designs/output.md

```diff
@@ -11,34 +11,46 @@ This document specifies docfx output file layout. It is designed to satisfy thes
 
 - **Static build output xcopy deployable**: should just xcopy static build output to any static hosting server.
 
+## Config
+
+The output URL schema and output path schema are controlled by two config items:
+
+- OutputType:
+    - `Html`: apply liquid to generate HTML output
+    - `Json`: output Json file without liquid applied
+
+- OutputUrlType:
+    - `Docs`: a.md -> `/xxx/a`
+    - `Static`: a.md -> `/xxx/a/`
```

Comment:
How about `docs`, `pretty` and `ugly` ? The initial names are `dynamic`, `staticPretty`, `staticUgly`, since we renamed `dynamic` to `docs`, I guess there is no need to include `static`.

Reply:
prefer to keep the static to make it more specific, so it be `Docs`, `StaticPretty` and `StaticUgly`

---
In reply to: [413447678](https://github.com/dotnet/docfx/pull/5815#discussion_r413447678) [](ancestors = 413447678)